### ISSUE TITLE
[Bug]: The home page shortcuts are linked together. Is this a deliberate design in this version? This greatly reduces the aesthetics.

### ISSUE THREAD

**Issue by nanmofeng** [Floorp-Projects/Floorp#1338]
### Pre-Submission Checklist

- [X] I have checked the existing/closed issues to ensure this issue has not already been reported.
- [X] I am using the latest supported version of the browser.
- [X] I have tried disabling all browser extensions/add-ons to rule out interference from these.
- [X] I have tried clearing the browser cache and cookies.
- [X] I have checked with a new profile and the issue still occurs.
- [X] I have provided detailed step-by-step instructions on how to reproduce the issue.
- [X] I have included relevant screenshots or console outputs.
- [X] I have checked that this problem is not occurring in the latest version of Firefox ESR.
- [ ] This issue is specific to this browser and does not occur in other browsers.

### Bug Summary

The home page shortcuts are linked together. Is this a deliberate design in this version? This greatly reduces the aesthetics.

### Reproduction Steps

Open a new tab.

### Expected Behavior

Reverted to the shortcut icon spacing of version 115.

### Actual Behavior

/
![微信图片_20240817173357](https://github.com/user-attachments/assets/8bb65d2b-5efa-4e75-ac19-30437c2f2cbf)

### Screenshots

![circle_screenshot_1_新标签页](https://github.com/user-attachments/assets/be0bdb41-93bf-4eb5-80d5-984757758aa5)

### Browser Version

11.17.3 (64 位)

### Operating System

win10

**Comment by surapunoyousei** [comment]
Yes. Intentional. It's the effect of a slight adjustment to the design.

**Comment by nanmofeng** [comment]
That's a real shame, as someone with OCD the latest design really drives me crazy. @

**Comment by surapunoyousei** [comment]
Should I put it back? :(

**Comment by nanmofeng** [comment]
> Should I put it back? :(我应该把它放回去吗？:(

I really hope to change back to the 115 version. I am an amateur UI designer, so I am more sensitive to these. Thank you for your work!

**Comment by surapunoyousei** [comment]
> > Should I put it back? :(我应该把它放回去吗？:(
> 
> I really hope to change back to the 115 version. I am an amateur UI designer, so I am more sensitive to these. Thank you for your work!

I see I revert it. Thank you for your feedback

**Comment by breadbeforerice** [comment]
新規プロファイルでは左側５つのメニューがブックマークアイコンの下に隠れて操作出来ませんでした
